tox.ini

```diff
@@ -5,11 +5,10 @@ envlist = docs
 skipsdist = true
 
 [testenv:docs]
-basepython = python3
-deps = -r{toxinidir}/test-requirements.txt
+basepython = python3.9
```

Comment:
Why 3.9 is important? The Ubuntu I'm using has only 3.8, so I need to spent time on some hacking to make builds work.